server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -184,7 +192,11 @@ func (r *Runner) Apply(ctx workflow.Context, root *terraform.LocalRoot, serverUR
 		return errors.Wrap(err, "initializing job")
 	}
 
-	planStatus := r.waitForPlanReview(ctx, root)
+	// auto deploy requests from refactorator bots
+	planStatus := Approved
+	if !isRefactoratorBot(r.Request.InitiatingUser) {
```

Comment:
There are 3 different things that can make this qualify for auto-approval now:

https://github.com/lyft/atlantis/blob/a777ccd48e0fbf85cfb75b8524f5fb8d1c5f8ec5/server/neptune/workflows/internal/terraform/workflow.go#L160

Instead of just adding it here, consider a light refactor to abstract this away from this file.  I would personally do something like pass in a list of generic auto-approval conditions and run through them, if one of them evaluates to true, then you autoapprove.  

That way you also don't have to modify tests for this file when we keep adding more things.  We can test these in their own structs.